ios/FluentUI/Label/Label.swift

```diff
@@ -98,6 +55,23 @@ open class Label: UILabel, TokenizedControlInternal {
         }
     }
 
+    public typealias TokenSetKeyType = LabelTokenSet.Tokens
+    lazy public var tokenSet: LabelTokenSet = .init(colorStyle: { [weak self] in
+        return self?.colorStyle ?? .regular
+    })
+
+    @objc public init(style: AliasTokens.TypographyTokens = .body1, colorStyle: TextColorStyle = .regular) {
+        self.style = style
+        self.colorStyle = colorStyle
+        super.init(frame: .zero)
+        initialize()
+    }
+
+    @objc public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        initialize()
```

Comment:
actually, it this a bug? what is the default style and color if you call this? without setting them?

Reply:
I don't think it'll ever be called since we're making Label programmatically. It was already implemented, so I didn't change it (I just moved the code). I'll go ahead and replace it with `fatalError("init(coder:) has not been implemented")`!